package.json

```diff
@@ -14,7 +14,9 @@
     "prepare": "ncp src/originalBudgetTrackingApp/categoryCalculationScript-example.ts src/originalBudgetTrackingApp/categoryCalculationScript.ts || true",
     "lint": "eslint --ext .js,.vue,.ts -f ./node_modules/eslint-friendly-formatter src",
     "lint:fix": "yarn lint --fix",
+    "postlint": "yarn typeCheck",
```

Comment:
I personally preferred to use explicit scripts instead of `pre`/`post` scripts (except `postinstall` and other special cases)

Reply:
I don't have a preference on that. Changed